source/characterProcessing.py

```diff
@@ -519,6 +519,26 @@ def __init__(self, locale):
 			log.error("Invalid complex symbol regular expression in locale %s: %s" % (locale, e))
 			raise LookupError
 
+	def _replaceGroups(self, m, s):
+		res = ''
+
+		escape = False
+		for c in s:
+			if not escape:
+				if c == '\\':

```

Comment:
Is there a reason not to use a raw string here (to remove confusion around the number of slash characters required)? 
```suggestion
				if c == r'\':

```

Reply:
This was already raised in another comment: that doesn't work, \ is still interpreted in raw strings:

>>> r'\'
  File "<stdin>", line 1
    r'\'
       ^
SyntaxError: EOL while scanning string literal

Reply:
Ah. Apologies.